lib/defaults.go

```diff
@@ -30,3 +30,13 @@ func GetDefaultBin() string {
 	}
 	return defaultBin
 }
+
+const (
+	DefaultMirror             = "https://releases.hashicorp.com/terraform"
+	DefaultLatest             = ""
+	installFile               = "terraform"
+	installPath               = ".terraform.versions"
+	recentFile                = "RECENT"
+	tfDarwinArm64StartVersion = "1.0.2"
+	VersionPrefix             = "terraform_"
```

Comment:
Going forward we'll need to define one standard for var naming: either camelCase (I prefer this one) or the PascalCase. Mixing these both introduces a tiny bit of confusion IMHO.

Reply:
Actually PascalCase Names are public in go and camelCase are private.

Reply:
Oh, my bad. My Go knowledge is pretty noob =) Thanks for the explanation.